src/PKSim.Infrastructure/Services/MoBiExportTask.cs

```diff
@@ -163,12 +163,14 @@ public void ExportSimulationToPkmlFile(Simulation simulation, string fileName)
 
       private void updateRepresentationInfo(IModelCoreSimulation moBiSimulation)
       {
-         moBiSimulation.BuildConfiguration.SpatialStructure.AcceptVisitor(this);
-         moBiSimulation.BuildConfiguration.Reactions.AcceptVisitor(this);
-         moBiSimulation.BuildConfiguration.Molecules.AcceptVisitor(this);
-         moBiSimulation.BuildConfiguration.PassiveTransports.AcceptVisitor(this);
-         moBiSimulation.BuildConfiguration.Observers.AcceptVisitor(this);
-         moBiSimulation.BuildConfiguration.EventGroups.AcceptVisitor(this);
+         //TODO. That should be done better I think. Can we accept a global visitor?
+         //moBiSimulation.Configuration.AcceptVisitor(this)?;
+         moBiSimulation.Configuration.SpatialStructure.AcceptVisitor(this);
```

Comment:
@rwmcintosh I think we can just use the one from the Configuration? Do you concur?

Reply:
concur